.buildkite/steps/tests.sh

```diff
@@ -1,13 +1,25 @@
 #!/bin/bash
-set -euo pipefail
 
 go version
 echo arch is "$(uname -m)"
 
 go install gotest.tools/gotestsum@v1.8.0
 
 echo '+++ Running tests'
-gotestsum --junitfile "junit-${OSTYPE}.xml" -- -count=1 -failfast "$@" ./...
+gotestsum --junitfile "junit-${OSTYPE}.xml" -- -count=1 "$@" ./... || TEST_EXIT_STATUS=$?
 
 echo '+++ Running integration tests for git-mirrors experiment'
-TEST_EXPERIMENT=git-mirrors gotestsum --junitfile "junit-${OSTYPE}-git-mirrors.xml" -- -count=1 -failfast "$@" ./bootstrap/integration
+TEST_EXPERIMENT=git-mirrors gotestsum --junitfile "junit-${OSTYPE}-git-mirrors.xml" -- -count=1 "$@" ./bootstrap/integration || TEST_EXIT_STATUS_WITH_MIRROS=$?
+
+echo '+++ Test Analytics'
+if [[ -n "${TEST_ANALYTICS_TOKEN_ENV_KEY-}" ]]; then
+  cat "junit-${OSTYPE}.xml" | TEST_ANALYTICS_TOKEN=${!TEST_ANALYTICS_TOKEN_ENV_KEY} bash -c "`curl -sL https://raw.githubusercontent.com/buildkite/collector-junit/main/collect.sh`"
```

Comment:
https://raw.githubusercontent.com/buildkite/collector-junit/main/collect.sh

404 Not Found?

Reply:
Ah:

```suggestion
  cat "junit-${OSTYPE}.xml" | TEST_ANALYTICS_TOKEN=${!TEST_ANALYTICS_TOKEN_ENV_KEY} bash -c "`curl -sL https://raw.githubusercontent.com/buildkite/collector-junit/main/test-collector`"
```